From 65b99b3e6aee94b1194a19117336cb6b8aea4c88 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:58 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Update kresd-keyblock.asc to the current copy at
https://secure.nic.cz/files/knot-resolver/kresd-keyblock.asc, the URL
the spec names for it (same certificate(s)).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 kresd-keyblock.asc | 347 ++++++++++++++++++++++++++++++++-------------
 1 file changed, 248 insertions(+), 99 deletions(-)

diff --git a/kresd-keyblock.asc b/kresd-keyblock.asc
index 1b34475..d5d1ec0 100644
--- a/kresd-keyblock.asc
+++ b/kresd-keyblock.asc
@@ -24,103 +24,252 @@ ofR/Zs/fvEvWqRXKm+I/qC1nK/glfaoA81xI6OGOPkrYEoLi9WKGK0nWSbnux9qY
 vm/kIQDhUsQmWbvIkwx6pHZoFw+9COY1NFINOqguBYzyMIlVJpo6Xns+1gldJ7vI
 0w6F+wpmUkFNnEACQ3dG/uf6WNzEbgEdKFQ2xgLN4zY7nJI3QLRlrXntk6BzG7Lj
 iQe7ZS4uhaQQ9bzTuvhi3qKJEJUGaPQRvYJoCoDQs1gLR/rep0qQpRsUt7Tzb0uZ
-xeeZHSvvhDq5uQINBGImJYQBEADaE8cZEkr06+TnlQ+8qSBHRFJSEunNKjUr8t0y
-zwPgFZhRIRtw3mCFqDEv/eBPw9tsQeW6aJz5+VI5t7Ifkt8krHPE8GT5N33MyUKj
-8sRk1oQNt/2JdDCOy5ZigyqFj31/9XeIWvlQbBLUTL0V3EPAmcipegEx9VFD81FW
-qH4TbBer0oLsa7zFnZ455FIn1a76UNC+73DE3+eV1V51euhx3TIgn76SXe8Qf+jg
-Asv7Ylfe++g3Smkt5UAgEZQOFwPbYMjS4U5sRq8hpJw+nBkI2n/vHoLeSKWZjt6J
-/yth+tLQmsQRwyYk97Tp8mzPpmWSMWIGPDrfyBdV1DmToQkWDozuBs5Mz8gVvrMu
-2mNe4O0m5Xv7e80Gm/OJxSeB/o8tEXFE12JEM1c88OZ8Iz/PcfSgotbcS5zVxRcm
-yU/ZqMRNvJHtG932DIhZ/s0NdAxnsAeCz9gaR6Z4orVXb8G7/S1pe0ab66k19VX5
-wFoPIz92SUq91VbrCsjfHi4FeE7h2OL9QP10Liv9qOxJFYvy2fvWG9EXuknjaZsD
-igmC1DuX/7uWJbrGbp9spnuNsgm7wYG5nFH8fTnNKtaMl2/D8D5WbB0k+MJnBDeW
-MyDKPsaCStX2ih90DGSOkWlZy1k3X+v+PTPPxu1ry/MePSo0UHud9L2oWLbnz27I
-bBRymwARAQABiQI8BBgBCAAmFiEEMFfumkSPNi10IFp3mrEg2gp29t4FAmImJYQC
-GwwFCQPCZwAACgkQmrEg2gp29t7Trw//cN9UEaN1WaH4jFDKIST+xdLnSLvk0Izt
-hHqJzzW9sRQYINvqz89EkTUrjelbW3Ib41YVNNk3S7hOm5OJm/fRHdQtItzuY/9E
-RafF5+PUiDj5TRDa8bRV7sCAQLoztfr1ozadbGE7e8HrrZv4DcZ10joCpLRZm04j
-QMBeZcaaEa2Wh7Xf9VW4aBJ/DU5DevsKeTRpCuwtw2lEf+NMl3PogPa9glm7liQu
-DKU4Zlf/31ZCudAb80PugOzWM2i1DG26G/b7z+8N59m6dBYWeSvcqU7Z9Q4rU7bV
-R/6nST8GyLIU/PsDalvKm2B3NDcXerN8lT9zU1QBzT3OyL6Uy/UzXl6YnpxFT8m/
-Gm4uiLS+gWD2TXh5BKGsuU4zIHakjglA7Gtb6PJ56+Pkz95apx7vAhmnJxgw7qAX
-KAH0Rng7IxEeL8Fj67tzWTQPs0PuAty7qIgbeft8hvhtk9vfFZb8k3bhX59b/+TO
-tbTOCjVhmq9bZdaw60SBl3LuyUlnlGmrqvPzVUdcAc2zvnxjEt9yVioS/M8zNRC/
-SmWbagZ/xyf2iiEAbo0y12t3heztdG8jMoMrImmUEubfPiAfCN2ys5zPiXW2jio5
-vvopiu23y37ew3zoAZrnfmFW0D8A8iD2g0kl/Ysb2dXIYocQ423ADN1Em/+1f8j5
-3P6DgfI04ROZAg0EWAOSdgEQAMcRDAsGV+ptULOruphwEUVNzVYIAW2qIp6E6uwh
-Ocq6xnDieFjCUpDFy6aCPTXEk6Ft8cEIF4aUPJ1ip6YqDgh9jafBLI/Bllgwmfsv
-sAasJ7K2vhUy8MnujuoNmOOKbiJ+/4ZTB2TemVfHqgwXidy16wjxoA64tQcCfQ9+
-sZ82FahqCU4qB04joMaOEj9oaS/L6Osj/5ex0xX7d/VG4Q58j5i7hJcWMnKECaoQ
-4FHHF0eoqt4ginPcuDfzJbCJUZ/CBBP6cUjP0GkcfeVaDW5LzEzd7Nx4GZFhXzNP
-0+9t5nnsGVpNHqstp7lFjBXf3s9IsGqouQHkfRMNFQUGfAGxShst5tgeYkI7J2/I
-2/kAWQuYtns7iPaMWjVY9Iy3dOEzrBHCqucHgaL7qifKtl8TUOdVkypvFndVdr4T
-hLljCt8ItQSsYSf53jQ62hFmZBeFE9UCx2OKYyP2gUz1GjFKBSsODohrefQisoVv
-77McAVuqRHmDirNeI4wQbrbUZgR7HawMQrP88PgJUqU0j16V3hOgfn3aQDDMmZ36
-wCL5F+izQ0TPG4V3R6QA55ahRGjTOhhNJY+Uw6YUy9ykfos/eV4mZjvHjP9vWeD/
-7tScqidUAtU8VtFsSU6Qh8C0spxQSh/BXWOvsPJgY9qUm2IMk+iYHTjVkcKPZHHJ
-dj+hABEBAAG0KVZsYWRpbcOtciDEjHVuw6F0IChwZXJzb25hbCkgPHZAY3VuYXQu
-Y3o+iQJUBBMBCAA+FiEEtgBkYLYKgOeCBiRJ50ffH5V1o6oFAmFmsBgCGyMFCRTR
-36EFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AACgkQ50ffH5V1o6r5GBAAxkNaB8zi
-TVWBZ5geqcQeoqCeTrZjDZ1pY32BbjefGb0z0fmRzH9ESQBGH+b46PO7cibOt3sY
-d4ZX8GxG/vnwsU3dOspaIQfHH99yXr+sCRkcr2k7DWTMFSX/5JkhlrouAURpIRN8
-y4+OO/TkWFY3HAIEy3VkhwIoJSUEDfSINb1KHnotNzHc7s1Z3yb14K14XV/LnQL/
-/NTz32NBGSDAt3C3+w4h5WKhg+kPcuno1VVUHM4AcqqLkhEbNN/QwQULWbrQMqcB
-+JJWlEgSqkhn2nrw/i8q+tlK7FBi4DhID99xRljt2HcPt7gqhnZ/2OxfoHGHui/f
-FPgmeK+c/wHZ22FH4ly0XwsSdIjskhRf/XPTaBlgeorfyIEBFBeFkgjdFUNSpl1M
-Tso6ZYxXkLg1UxEFtI+X5Eh7zpirfs6jwc44aMHKabJY3V/MBmYqUBRR3H6xnIFd
-gfjVeFAR65w7MElJXRx2kvJ5ixpV5W2+U//MvhkpN6Vk1Y2G9NXmQeGXz2l+2A9V
-4qpTn/nVAIchYFtVPzsAJlyPAqB7I45uNMQOcsGdn4W5lLgD4MNdgwIUce9jQ1Oo
-F0BnZSCJGLjU0ZxVUOd7fwF+P+Y8RBxBIhVcvxaluI3YmT0roqCXaJaYGAWgXZCF
-9g0NnfvR0dRRZRUeT/qLxJNZksK+iFtLIlu0MFZsYWRpbcOtciDEjHVuw6F0ICh3
-b3JrKSA8dmxhZGltaXIuY3VuYXRAbmljLmN6PokCVAQTAQgAPgIbIwUJFNHfoQUL
-CQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLYAZGC2CoDnggYkSedH3x+VdaOqBQJh
-ZrAYAAoJEOdH3x+VdaOqVjIP/1N5v2soWf7T2OcromFmL76QDqAIwzYtuwookFK8
-UBiGEAw+lz8dHSPqiQF8IiaVxroWeKp7vHKjSeRCr2a9eKsYzTdz3jStwPnKZGpP
-5OMJb2x5dZNxyKaL1Fyidq0N9LDIkxanMORPYVA2c1tHmFJFJoj8EmUhIYN8iAl5
-ZG2u3vELI9B3ZvNJb8OJUxAOcvPVyuxKMzpJY2C8K4Ec2joLugG+aCoZh7DTi4EO
-tqAOxr/jfCPEdPHEIKEOdqXy5DtoiA5e6uHA+DpseBC/jM2jrvO+IRqnFvr7M7+Q
-yfv0ZbkFefsJp5tjojCv6xJPtz/BKwh9wxNtUJrZfFL71Otz4/0YAwiX4z+kljop
-x0i4ALVUOiWb7i/2PiP23n4JnPAfLdlClk9CeTkCOu0CAgBvrKi1w/7eEnGqOgTu
-4OaGfFaOtTjuFqKS0wOdBRGST5h3pKjxrDxN8HgI+ZpF1EfMd6ovfLWpMVIwE9cV
-ueimcpAqfSubQECTjwnsDSmzc26apaCHL8FaXf2pG9J5uDczxdcUzVWfgb+16Wdg
-bFoqJ82O2SCysTcEkrNH8m2R5i7iCaO5WlQFLf2FHOBWLd3RjV1ztCImv4di5skw
-QLwgW2+wi0QowVPgPc38JAPa73INRhaPK3JUcwqNEr9JUNfRGyL+eTwTX4b5YKHd
-dJ9KtCRWbGFkaW3DrXIgxIx1bsOhdCA8dmN1bmF0QGdtYWlsLmNvbT6JAlcEEwEI
-AEECGyMFCRTR36EFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQS2AGRgtgqA54IG
-JEnnR98flXWjqgUCYWawGAIZAQAKCRDnR98flXWjqjpMEACClbUGJ3e0p1Nty0Lf
-kRsoWZMoyeWSoBA0owUsQOyquKKaSj+40qHMDOhbNfJoLpwFsU5zJL4I2rhDlW+z
-Y5Het5x1afS0lExzXK3RMPWwhUwMmWevUpW+VFjvPyBhdlEP8E0Ayj6z/p5GEN9Z
-9yruZfz9ru48KDvwUYz/f2V0cAPqOIcSM6euGLXnK4ExML+8Wiqgmht8qBozD0Lo
-odyAWMR1D+XkAMR+ugwUfs449+x7zzzfkRh8CfggB9mgMK0i2b66MdWFHYW2c9jc
-3SIEaWZM56pEG9BCZpcoiiGgCOMWQ7CEn2dOfH2Ah1ZiICyDnL9XvESYJBoRLiNG
-cLrD31GJzUrDPRLzK9f8j4AlWN7bNECw5pmPYE2vizFOuasojBV6hlg+c0bXpQSK
-ybg1eQb0rE5yOWrA0pP5yNc6ZIBxtjbYLbwmWltb9mWMznVSNfcjvA7QXe3srYYL
-AOW1KPz2Ly8qCTZXcpXx2Z2mk4FTNDIj4maQxLTIURHWHckHrXxENxMXNcbaZ4d4
-LVz5f1RBhsFfmAXpVu/FTvXIPlIWaNVlQDWBMxlVidpeztB4XgPUqKWmtMNyCTsN
-PeuxrRiyIm2ccXE+fcuPDstLIeQdX+U/rijx1f588i6nt2oD0vy9tEchFeiczql1
-YzYNM5U8u/rCYiOxmQWf7gZecbkCDQRYA5J2ARAAyHww3huLEtsdyqgjiGMhtEKO
-Lmp7yFl450HY9oPcHS02U5BC1370ssNShrdOCi2ACDbe41Zxx85WcuaO1OVqung2
-umX047mj2xQsiTAFRDLZsQu8cQFoEy/DBL2bk7ThfK1Lh+NyZAs0UaPpDkGodS0D
-e9osA+4T6Nf4POYaeavbYVFSdDKS4lUboBqApKnD/TzKFxFcpuFx6FN92lteTbOo
-jGMiLoZvELY86Kn9KuFZ8FM2ZSNHx1Z75KouufGrdkeCoZYVYiuzT+fnt2it4dIp
-IlnF+yxMt5LB/MSrmECB5CAFJtxzuMccm6yDUZQSWWi9vUgxIJwvt5w0CIBT353D
-GeP4WnH0r5YoBKoRbh7i4fT0lWvMXTG/V2lqyzBdClMebyHffMgba26Kj6oeDygD
-fC5aGsVaqw1Ue/qQ5QRqTJcJV7xVLTtS1EamVqkfKwPS0zTfnrF1jQtnO/P4qkfg
-BRRG9BXGGrykHpXOyqmX6Z0wbV2P4j+p02oSecDl5yVXplJfsXfbS/xXnaSkaN/7
-mCU29ul26cAVNxDkDPunztSFi9K9LM2T/XWYJQGXM71OpmONQJGF24lx7Wp/kobn
-HtbjGDzjDPC4eSL7MA56qtrWaLM+4ePKANct2q0q6c0uSLs0Q2zochS64Mcg0YzL
-1sinWPN1rXLDk3lwpIsAEQEAAYkCWgQYAQgADwUCYWawGAIbDAUJFNHfoQA/CRDn
-R98flXWjqgkQ50ffH5V1o6oJEOdH3x+VdaOqCRDnR98flXWjqhYhBLYAZGC2CoDn
-ggYkSedH3x+VdaOqtBgQALeChWgHEGe/8nwKWrC8CMQyKyiJRlSfvERi5M40PYxw
-KC7IHo+ekMdlLc8kVkv5WSq3zQcNoRMjHutZlCVpGJat5PwiCine6I3Z5JfKsU7f
-JE4KXPD0jr9kCy/IHlCWsKLZTHH0LOk98tqZcFQwFM5nQEzu8Us86BxlZs5IaN0j
-ILD2qUC+EYeg0hd0kya+16UNao9NxCeUEoSwxhtB4IIMCJTUPx0pWErCHxeCUjvx
-Pd2h5A106DT/3fa8uhPHr9goHtfreV0soGuxu1rqGrMn7XcorsGdt9XX4tyPHcXD
-XCZMLedjYTu7OcYQ63hi+ZWUXJPvcIrB4StCumaFECwLodpdDWB7n6OF8r+X//eP
-bjS8qt++A20LGD4l7BrWxDeymrnqLmWN8RBf2xV4ytVfcRx1ercFMGiNRUvGSh+k
-gP6J7D4yT4xGTq+fCtS6BHyWMB0loXcWv0Im/6znpIpUcvXyL68s4Jiqukt94rKl
-wb/IB6MSWannxk8UzWfGgFXeGUrR8YWLuQSo26AceNOyu2gMW+k9kYi0sqkBzMPk
-1zWZN/gSNpDyu7AWGpv4RtOKhJhaux8zwtwqozqSx+plpnYz5ifwCNMuznM6T0j4
-msKQiK1N7lIwKBGaT2P8SJ/ecaJyudIz4ds2+GUl6fxqSR27Egs+EUY0BmS7wESr
-=dRuR
+xeeZHSvvhDq5iQJUBBMBCAA+FiEEMFfumkSPNi10IFp3mrEg2gp29t4FAmXllIcC
+GwMFCQeBQmwFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AACgkQmrEg2gp29t7s0xAA
+lc/bQ6+8iTkwwStPkiItlNNtceyIcFzcxnpr/sQF9ZlQ30Jv5FriQIh2A1t0sLCR
+CcFheKuRfjaQnKyYpcxzaIhpf9BKmJO+sSHChGkv7n8pFSWAbr/hnOFZBL4ysN/2
+L5WfGnx2uhp+u+VGlgRsXuWdvi3tOZwI2dcPRdw0uosMLz8+s3yIKdAhI/Uh3yhN
+4qzJSF3h2hJYYVGeuiv+EtUTZvhg6nV77qUwtiRe3+HlMlMLVpwDfONds0sZNi/I
+ZOVe4TZV2idFAQlE1QMqX7ShUKj0PBseOgX17ZsFNqJya2QCPyhsmqPFyi3z92St
+g/BRyAskT4urNnzppoLxcPMBkrRSKSBE+omO3LvztZFfbaVI38a6PR7ROWt0PwgH
+cT9Be+oDDAh+nKTTKtlgHbPB4kJo0u+MqZE4v7NdHvREkkJ9k4qGX5WggGTMQSiV
+kdQCrhQmMg7FE+cdQola66qox7pT6a/ScPzhYWVQYFZRVB6CgSQcWhWu/PJpqxmc
+FkGKZUEgNtfDkRXEpID+LljcSsTN7AccPJVEfZV9XDKeZuh/brUcUYjCzo7pG0QL
+CKJQCB7WovMP6Wl29ftrwVa7+7pAkLYFOhP6KQWkE68XJHVtiigDXEyZbwJfaeb5
+ltVaGMUWp0Orcb+4ytcyyhQXFkv1jZ9pEQzoS1vYWQ+JAlQEEwEIAD4WIQQwV+6a
+RI82LXQgWneasSDaCnb23gUCaYNefQIbAwUJDuJ37AULCQgHAgYVCgkICwIEFgID
+AQIeAQIXgAAKCRCasSDaCnb23sq5D/4uStDTwdkLhV25akCSSrWGu82jcqDw1Dx5
+h2gN3kD5Xnc01C0Robs9kbvRxzne3mr1CE7bJYZDP9CRsMDbMSCyhT5omG0w0sEq
+DYHTy44Yfe+/6Ab7X9e5Mjb0lNYMCMOUtaKzzFi5vgGgYxnN1Fyfnz1vTQXj3h/y
+HlNPzVD2ApCpG0vCFrSSyGIs8gMMDWgMfPU3HT4J9sRSlTrGsHWHAU1ghnIrPR90
+T1pUWOIucCjS9mO0ydliwxsyI/Yx+0l11q3r5hO5zKtjUbbb1AIikEbwz3b2u1eh
+UHGIGOXhXYPcMbzeRFv8YH8EYb2J0e9uaI6+Clmh/kwi3vQAun6pfxxRsNYwcrJR
+/93wxf85vBMOB9Qeh+MwSkTT9+UBaHMn4L6sf/5rezTlS3izFDZKZ4wEQV09YyKo
+zvOZLZ9sAkRFYlw4VQ1jvOrB+/Ot8BScdDCtLAQAypGjJMyYkEIb01VhWeml2Rwh
+EO0G/HWdyigX46gQZAhs4wSZ3RDtTRppBmdItJCG/qq10qecs5QfSaBOgNhRBmJH
+GLncaVUuYxdEyO8wTiAMTCd2T+MIn9qUMk1eKmS2Mzfcax2Dcm4mYik4WOlzD1Oj
+wEv/LRZXhXwERnGSlka2J+ghQLe4Zp57FEcqN1yaBaMvl/XTOE7EIcSagsfQpyWV
+VVzOsTnsRLkCDQRiJiWEARAA2hPHGRJK9Ovk55UPvKkgR0RSUhLpzSo1K/LdMs8D
+4BWYUSEbcN5ghagxL/3gT8PbbEHlumic+flSObeyH5LfJKxzxPBk+Td9zMlCo/LE
+ZNaEDbf9iXQwjsuWYoMqhY99f/V3iFr5UGwS1Ey9FdxDwJnIqXoBMfVRQ/NRVqh+
+E2wXq9KC7Gu8xZ2eOeRSJ9Wu+lDQvu9wxN/nldVedXrocd0yIJ++kl3vEH/o4ALL
++2JX3vvoN0ppLeVAIBGUDhcD22DI0uFObEavIaScPpwZCNp/7x6C3kilmY7eif8r
+YfrS0JrEEcMmJPe06fJsz6ZlkjFiBjw638gXVdQ5k6EJFg6M7gbOTM/IFb6zLtpj
+XuDtJuV7+3vNBpvzicUngf6PLRFxRNdiRDNXPPDmfCM/z3H0oKLW3Euc1cUXJslP
+2ajETbyR7Rvd9gyIWf7NDXQMZ7AHgs/YGkemeKK1V2/Bu/0taXtGm+upNfVV+cBa
+DyM/dklKvdVW6wrI3x4uBXhO4dji/UD9dC4r/ajsSRWL8tn71hvRF7pJ42mbA4oJ
+gtQ7l/+7liW6xm6fbKZ7jbIJu8GBuZxR/H05zSrWjJdvw/A+VmwdJPjCZwQ3ljMg
+yj7GgkrV9oofdAxkjpFpWctZN1/r/j0zz8bta8vzHj0qNFB7nfS9qFi2589uyGwU
+cpsAEQEAAYkCPAQYAQgAJhYhBDBX7ppEjzYtdCBad5qxINoKdvbeBQJiJiWEAhsM
+BQkDwmcAAAoJEJqxINoKdvbe068P/3DfVBGjdVmh+IxQyiEk/sXS50i75NCM7YR6
+ic81vbEUGCDb6s/PRJE1K43pW1tyG+NWFTTZN0u4TpuTiZv30R3ULSLc7mP/REWn
+xefj1Ig4+U0Q2vG0Ve7AgEC6M7X69aM2nWxhO3vB662b+A3GddI6AqS0WZtOI0DA
+XmXGmhGtloe13/VVuGgSfw1OQ3r7Cnk0aQrsLcNpRH/jTJdz6ID2vYJZu5YkLgyl
+OGZX/99WQrnQG/ND7oDs1jNotQxtuhv2+8/vDefZunQWFnkr3KlO2fUOK1O21Uf+
+p0k/BsiyFPz7A2pbyptgdzQ3F3qzfJU/c1NUAc09zsi+lMv1M15emJ6cRU/Jvxpu
+Loi0voFg9k14eQShrLlOMyB2pI4JQOxrW+jyeevj5M/eWqce7wIZpycYMO6gFygB
+9EZ4OyMRHi/BY+u7c1k0D7ND7gLcu6iIG3n7fIb4bZPb3xWW/JN24V+fW//kzrW0
+zgo1YZqvW2XWsOtEgZdy7slJZ5Rpq6rz81VHXAHNs758YxLfclYqEvzPMzUQv0pl
+m2oGf8cn9oohAG6NMtdrd4Xs7XRvIzKDKyJplBLm3z4gHwjdsrOcz4l1to4qOb76
+KYrtt8t+3sN86AGa535hVtA/APIg9oNJJf2LG9nVyGKHEONtwAzdRJv/tX/I+dz+
+g4HyNOETiQI8BBgBCAAmFiEEMFfumkSPNi10IFp3mrEg2gp29t4FAmXllIcCGwwF
+CQeBQmwACgkQmrEg2gp29t7dkg//TIKucPoqGNikDyA1sNdx/ErdGH6GaXIS47AI
+8BNS0cvUqFRUufe5nxFZ7F4+JRUWyFMrg5OBLM0Q0+4iFHkt2tmRMi5GTSfE1BHo
+OaUfj+YKz+qN6w3/vqhnii7TaYy9CJdjmfmoXE2ow3UBUUmy+12OgH4lpyZfseGN
+iUZ+xTmrDj+N7D5BrgISyVRyPvfY45l5m8Fa+i3iD5Ek90KMQGxXMc1Rcyeej0n6
+aV6dZ1Y72jRTer+Z3tYIWqDPjtxZAnh9J7TeIRVCBOkFaliai/cHKyOWV/WSV/1l
+Sk7EeSfTpl+PMxzA72H5S1mYKGcwPkz2GziBLfExEjeKjh3zU2JHdquzRh2WleKk
+rn4YkV4H9uoKwfCwkupnoCinUQA07Ejye+WB3k63Pb/YRMCTDXw9yji0WsoYN120
+Y6NgIZvM9n0f4Gm6Af2RDgf2P1CatFaZf+yXV2EQnAIWBJgRe2xiPt/XAa3YHu0n
+8aS3vzVoXk62d8WzvVSi0N7KGp1nQlKkaumZOUeQ3u+Vo+MEdISFprLgYvz1mOto
+45mAK2ev6cr+uWh/oiC2U67Q+h11ytebF6yqNhvarKfKGWWFhgIWiXRKd6ULuRud
+sQCUBKsL6qSpSvnG6kfgDNZd+NtE7vIrv5vV1Q8C7CHumkVPjczng4aGYYgRrEGD
+pJK0g4WJAjwEGAEIACYWIQQwV+6aRI82LXQgWneasSDaCnb23gUCaYNefQIbDAUJ
+DuJ37AAKCRCasSDaCnb23m0ZD/9NPNr9+a+2xf1OO+HCRTKp6f9tl2spM5rMqm2E
+N7YpXFbtsGt9JAG3XxGgZ1QMgdpGV4+k7642heGspT2ybmRONmzpDp3YM0LIBTAQ
+QboevKN7Lwypco9jYJj3K3nxoeSKTwSSCB3jZBj9JBZLhlMeJUGqRxFbCENBHHpD
++sUjgPzD7yaAteUptD84gB2azEc48QV3jstPzSVRQFM6g+TBne0w6l24ACliIAWg
+FwqYV8Rhiom1AwY2+4+Y0TFaoCTnUb5915Onn1eUh8Rghw/1h9BbDmDtOegmm+Oc
+KCz/oMHRU/9X8e+yRAxyM1+gc6D4x8+2cimE+sMo78enGBPh3atW9UDonrQ373N5
+mIMUKKoypOvnfnAUVezWaYHmoIKkINzuBZcUxgSLis39jbLtsFnlsAbPdWjp6oMp
+OEMcNYnV8IB8f7TGl7vG6NqVo6q4Eij8KFUn6MdC4NG9wgQA3y3X+IPxZcmWxR+7
+q/Us+NxH14yINDfMxz8HyhpHg+eX+jsrZkZWKbZCVeBNqUqofH7E5Jg0ZeGNjYoJ
+VZDRNYQNcl9Tv0LgospHcmA/nS2ZsW+hYXuxCT1tPUJRdW41Iq3JGu+Bvyd/bAhX
+XAneeDargN9/dds2T+utxi50OCh3BtdeVg/FuWikSLMm1FAXps7XvxvHbGIYwpEz
+iTPT9pkCDQRYA5J2ARAAxxEMCwZX6m1Qs6u6mHARRU3NVggBbaoinoTq7CE5yrrG
+cOJ4WMJSkMXLpoI9NcSToW3xwQgXhpQ8nWKnpioOCH2Np8Esj8GWWDCZ+y+wBqwn
+sra+FTLwye6O6g2Y44puIn7/hlMHZN6ZV8eqDBeJ3LXrCPGgDri1BwJ9D36xnzYV
+qGoJTioHTiOgxo4SP2hpL8vo6yP/l7HTFft39UbhDnyPmLuElxYycoQJqhDgUccX
+R6iq3iCKc9y4N/MlsIlRn8IEE/pxSM/QaRx95VoNbkvMTN3s3HgZkWFfM0/T723m
+eewZWk0eqy2nuUWMFd/ez0iwaqi5AeR9Ew0VBQZ8AbFKGy3m2B5iQjsnb8jb+QBZ
+C5i2ezuI9oxaNVj0jLd04TOsEcKq5weBovuqJ8q2XxNQ51WTKm8Wd1V2vhOEuWMK
+3wi1BKxhJ/neNDraEWZkF4UT1QLHY4pjI/aBTPUaMUoFKw4OiGt59CKyhW/vsxwB
+W6pEeYOKs14jjBButtRmBHsdrAxCs/zw+AlSpTSPXpXeE6B+fdpAMMyZnfrAIvkX
+6LNDRM8bhXdHpADnlqFEaNM6GE0lj5TDphTL3KR+iz95XiZmO8eM/29Z4P/u1Jyq
+J1QC1TxW0WxJTpCHwLSynFBKH8FdY6+w8mBj2pSbYgyT6JgdONWRwo9kccl2P6EA
+EQEAAbQpVmxhZGltw61yIMSMdW7DoXQgKHBlcnNvbmFsKSA8dkBjdW5hdC5jej6J
+AlQEEwEIAD4WIQS2AGRgtgqA54IGJEnnR98flXWjqgUCYWawGAIbIwUJFNHfoQUL
+CQgHAgYVCgkICwIEFgIDAQIeAQIXgAAKCRDnR98flXWjqvkYEADGQ1oHzOJNVYFn
+mB6pxB6ioJ5OtmMNnWljfYFuN58ZvTPR+ZHMf0RJAEYf5vjo87tyJs63exh3hlfw
+bEb++fCxTd06ylohB8cf33Jev6wJGRyvaTsNZMwVJf/kmSGWui4BRGkhE3zLj447
+9ORYVjccAgTLdWSHAiglJQQN9Ig1vUoeei03MdzuzVnfJvXgrXhdX8udAv/81PPf
+Y0EZIMC3cLf7DiHlYqGD6Q9y6ejVVVQczgByqouSERs039DBBQtZutAypwH4klaU
+SBKqSGfaevD+Lyr62UrsUGLgOEgP33FGWO3Ydw+3uCqGdn/Y7F+gcYe6L98U+CZ4
+r5z/AdnbYUfiXLRfCxJ0iOySFF/9c9NoGWB6it/IgQEUF4WSCN0VQ1KmXUxOyjpl
+jFeQuDVTEQW0j5fkSHvOmKt+zqPBzjhowcppsljdX8wGZipQFFHcfrGcgV2B+NV4
+UBHrnDswSUldHHaS8nmLGlXlbb5T/8y+GSk3pWTVjYb01eZB4ZfPaX7YD1XiqlOf
++dUAhyFgW1U/OwAmXI8CoHsjjm40xA5ywZ2fhbmUuAPgw12DAhRx72NDU6gXQGdl
+IIkYuNTRnFVQ53t/AX4/5jxEHEEiFVy/FqW4jdiZPSuioJdolpgYBaBdkIX2DQ2d
++9HR1FFlFR5P+ovEk1mSwr6IW0siW4kCVAQTAQgAPhYhBLYAZGC2CoDnggYkSedH
+3x+VdaOqBQJccncqAhsjBQkJZgGABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAAAoJ
+EOdH3x+VdaOqXPEP/RBD60xMLKHsMfD2PhzcbsT+4hmvO9gJQEGgmWIOqHp2D1x+
+cAk+Ck8VJb3uuMa8qzi7RTKGxWQWrmat/C1mq1vPDLSIgjTEe+eTD64BsXJgo955
+5dMgQLUvD8JorsrAJHYQj65/JWKAQr3feZsYB96XtKLkl892vqvgIt6DSzMazyho
+H8rQRqUbWvd5O3ibB0AeIYIHgRsbwhM418eOFj8xAbx5EROdNC0x+EV7q4HJe0gm
+fJ34a3SWUxNtYfXPFF/H1N17k+oiQ68FbLZMnLTJ1q5MKCSrIRS0WmyzqzGZH2jL
+viS9FwzxauBeXOg353vjRpc5zWFWjGzczdY88L8k7jxwKYLWTDhTU2mmMhAHMyFf
+4XeeHjJ/kfFNDSXWW2OOAhEHM28nmBpIvHddYoMBjXGc4KmSf1aJlUFfTwrdOMA3
+WbooPKSXLalAULK7owkNmyZDrngTSIY0yPMRYPIArIOOmMpCO/ilStsfThQhj4uA
+I3SumQvT6T37MvmY4keD7eGwdOrQqmhbyo5J9hnPrQD+GR9eSXf3CPMV+4217BEv
+M+arbaDwPW5sYjWox+ED88wsq4rOK5CR0+pu6JS2bPQW+VCSy4VyOVIlj5nAnzpI
+ec9R4pE0Z6NJTZX9NmcpRZFn2q+bMAiIoMPPLSKCMLvMQh+G6fUcftpN8RAwtDBW
+bGFkaW3DrXIgxIx1bsOhdCAod29yaykgPHZsYWRpbWlyLmN1bmF0QG5pYy5jej6J
+AlQEEwEIAD4CGyMFCRTR36EFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQS2AGRg
+tgqA54IGJEnnR98flXWjqgUCYWawGAAKCRDnR98flXWjqlYyD/9Teb9rKFn+09jn
+K6JhZi++kA6gCMM2LbsKKJBSvFAYhhAMPpc/HR0j6okBfCImlca6Fniqe7xyo0nk
+Qq9mvXirGM03c940rcD5ymRqT+TjCW9seXWTccimi9RconatDfSwyJMWpzDkT2FQ
+NnNbR5hSRSaI/BJlISGDfIgJeWRtrt7xCyPQd2bzSW/DiVMQDnLz1crsSjM6SWNg
+vCuBHNo6C7oBvmgqGYew04uBDragDsa/43wjxHTxxCChDnal8uQ7aIgOXurhwPg6
+bHgQv4zNo67zviEapxb6+zO/kMn79GW5BXn7CaebY6Iwr+sST7c/wSsIfcMTbVCa
+2XxS+9Trc+P9GAMIl+M/pJY6KcdIuAC1VDolm+4v9j4j9t5+CZzwHy3ZQpZPQnk5
+AjrtAgIAb6yotcP+3hJxqjoE7uDmhnxWjrU47haiktMDnQURkk+Yd6So8aw8TfB4
+CPmaRdRHzHeqL3y1qTFSMBPXFbnopnKQKn0rm0BAk48J7A0ps3NumqWghy/BWl39
+qRvSebg3M8XXFM1Vn4G/telnYGxaKifNjtkgsrE3BJKzR/JtkeYu4gmjuVpUBS39
+hRzgVi3d0Y1dc7QiJr+HYubJMEC8IFtvsItEKMFT4D3N/CQD2u9yDUYWjytyVHMK
+jRK/SVDX0Rsi/nk8E1+G+WCh3XSfSokCVAQTAQgAPgIbIwUJCWYBgAULCQgHAgYV
+CAkKCwIEFgIDAQIeAQIXgBYhBLYAZGC2CoDnggYkSedH3x+VdaOqBQJaDfDeAAoJ
+EOdH3x+VdaOqYPoP/3pThwNkGsdDkuGrJuhxmv83YhLlavZIssHiTYXn9b/BsvTm
+Crodr3/ZMypejFGSG6D4QAcUfHKU9HjfI32vPSmMFceutrzsUA8cyqr+qqs4RR2G
+TK0SprpYiGhqgSR2FoyFTJy2NWMvZZnT4xZ5GOp50tBf5UKs5U9xVsut5FGED03c
+o+A3Bun3MKDqEQ+LiXbNkp50eRM2MogDAnTFmgzAVrVj68vwSbw6v4gFm4YEw/9k
+y1qw84hBpP8a+zLHUMOivCygJ1frNpNnLkM5Y/cUiszO6z9gNvPABjFKlaT+MXjj
+JJucnN2NTrsR8bdSuwr5FEbDxomqzf/iocpkxlmc0v7jAmkR2dnIWJz8xdzDoUGz
+NATCsg5ozVYR/DOfYTFRiTakznNoHCrvo6CuYYzMwSJx8wPKouh3ARCpHvQ+9FuS
+uH5HTYumHBD1zkAVdysuMoyZ5CVtBVqtNSysQtxMrqsSULWiNLszfuPRfMUT6SA1
+afZ/IFGC2cYNYRruv6gNq55cmcR/T/hbDbwjDFqQ1zuvQ3jUYgF6TBLEp2R6koVm
+C1UYZIOUV6LmFvkstYWWMDU5LsJFUkqlsdp9CN091vW82pkCPjLZIle9VWkjUo7m
+Sr50V1iKk5G/Q3YiGjow1htBRPUQF5Tjobk3S+IKgtLXn8/EuEgCj4/Mqlb3iQJX
+BBMBCABBAhsjBQkJZgGABQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEtgBkYLYK
+gOeCBiRJ50ffH5V1o6oFAlnIz+wCGQEACgkQ50ffH5V1o6pvJRAAkxE4HWvOUgKZ
+2hgeXQc6QhX+HBi5QfPLgC5bC1BkCTcbXuPdKuZZPdg5J/+0GGbtqWSJnXWXL1D1
+VZWs6lpgM9Aclsbn0XnsHSX4ukyafrR9v17CnUMMXgXoggTs1a3jd4xva0QDQq9w
+Dru94cZU6rsBsjimeNvfRkiG6buAHqZJPblCQ6VDI9OPopHpk+rV93Bj3gLVnrbJ
+AKNNHljcxFRV7D1XC19aFJjrGjEIYBCvxK26Mg3VYMm/jo71nayFWyS6/ZaEPTCV
+QCqrgX6+4ACXo/tMTSyFQ8yUMXKmqMZXNKynjl7KeRL08iC2sZ4B5e7ceuu95xv/
+4DzImQf5ETBbASxiyrSHRRb60tDskwjh7pFxZGHW0G/HAT3Ss3wPbksPwxsMTIfg
+OLBqSiHw2ZS92Y3DQbU1KH1XWSYdhwCAm9qF3kKZyZYOiKDI9KZ3wbF0e7p29cvk
+mJ8DbXtxmLps7J7w5Oe5UIqVHr4GYovtSuzRS+AxUEpBWeQOnxKNe6/oEHmuLGdd
+HcE6U8wUGGuz86gnhU+r9tfSkA/aFUcdBIiIiqYSzMpqjqNGKD727k77DbJzbuOj
+CXF+cYiyCK/KvyYc3XMNK71XI7y9valWbt4zNyjgkyKjiZScCaHv0A7LDalF7gOh
+spcevd0dapstkMv/HvJg890qO7Ox6NyJAlQEEwEIAD4WIQS2AGRgtgqA54IGJEnn
+R98flXWjqgUCWcP6KwIbIwUJCWYBgAULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgAAK
+CRDnR98flXWjqil6D/0dSV7kphVFpx1mrqdWGn16yTRK+xMEz8LuohbMnSmqu977
+r8EtUj3p3z/MSlP1g3v/Om1GCfNl1I74Ih8Qb9cyZOD5wkavb6rXUqbVj/gkYahk
+GWHU9uoP2b2diIKMMi3kVmSOSh3z9yrM9RXmeWqECAeoe8knNydug4CZ+X01qZ9Q
+JrC6Vf0c4Qo/r8n4GDCdG+4cLAFqIvfOn7mZbLjOQGJJ+8PlhHsBPCF5VXfGM+et
+rq2myHAFismsjieGhhBOrd6JoI/+v7wDEkjElxVh0HrBrT9WDdM8ZuzrNS6HxmgK
+H37C+CR+nyInX4I8Urghl2Sqz1P4tX0Rg2HO4sDN/5R5SFPtm1sycE0dRHtS0OGD
+paCoN87ylgcLw8KHxpFZbWBA8FeyW/nYWOnLH+BU5iq0wCksXTeutG7seGciGuIC
+1on/IlemIjAqc4/niIb0tqYQQ9PX/dkInTN4mu29v4eX/0EDceWiPC5cHLtTBjKa
+kU7Uo9mUdRkrkUJCLBN6vdD9rr3uvQZ1jHxlpaAVN+oMAElGQBXB+8Zvwpaf4/Uv
+rZqXWvMSgV6DglfFoFF8pGvDUBKHjJhukvz9GHSPN62FURucrfrOAKkuumppqQXc
++EXr2YOrF4PItpX9VEk6Tz7T8Z2d9nagxNMRMf3934Yp7O9i+1h+MDaa719AAbQk
+VmxhZGltw61yIMSMdW7DoXQgPHZjdW5hdEBnbWFpbC5jb20+iQJXBBMBCABBAhsj
+BQkU0d+hBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEtgBkYLYKgOeCBiRJ50ff
+H5V1o6oFAmFmsBgCGQEACgkQ50ffH5V1o6o6TBAAgpW1Bid3tKdTbctC35EbKFmT
+KMnlkqAQNKMFLEDsqriimko/uNKhzAzoWzXyaC6cBbFOcyS+CNq4Q5Vvs2OR3rec
+dWn0tJRMc1yt0TD1sIVMDJlnr1KVvlRY7z8gYXZRD/BNAMo+s/6eRhDfWfcq7mX8
+/a7uPCg78FGM/39ldHAD6jiHEjOnrhi15yuBMTC/vFoqoJobfKgaMw9C6KHcgFjE
+dQ/l5ADEfroMFH7OOPfse88835EYfAn4IAfZoDCtItm+ujHVhR2FtnPY3N0iBGlm
+TOeqRBvQQmaXKIohoAjjFkOwhJ9nTnx9gIdWYiAsg5y/V7xEmCQaES4jRnC6w99R
+ic1Kwz0S8yvX/I+AJVje2zRAsOaZj2BNr4sxTrmrKIwVeoZYPnNG16UEism4NXkG
+9KxOcjlqwNKT+cjXOmSAcbY22C28JlpbW/ZljM51UjX3I7wO0F3t7K2GCwDltSj8
+9i8vKgk2V3KV8dmdppOBUzQyI+JmkMS0yFER1h3JB618RDcTFzXG2meHeC1c+X9U
+QYbBX5gF6VbvxU71yD5SFmjVZUA1gTMZVYnaXs7QeF4D1KilprTDcgk7DT3rsa0Y
+siJtnHFxPn3Ljw7LSyHkHV/lP64o8dX+fPIup7dqA9L8vbRHIRXonM6pdWM2DTOV
+PLv6wmIjsZkFn+4GXnGJAlcEEwEIAEECGyMFCQlmAYAFCwkIBwIGFQgJCgsCBBYC
+AwECHgECF4AWIQS2AGRgtgqA54IGJEnnR98flXWjqgUCWg3w3gIZAQAKCRDnR98f
+lXWjqs7GEACZlVtvy0Q45DrRQJ2B5SAeb0ZJ5OZQFPFnnl4UjL2Q9A1jglzjftbh
+jfwf41K9ouUoa6R8X8nlpGwo8DSZwXNYni8AXUMYh01VgSFop/6Uxeaczyz+X6/Y
+J5Q+UMEkVz2rrezpZXG8pj0+yf8fGbImEqGDJInQZoJhUDaaFSiyFIMJWQUE52O1
+17fAUvDDfVdvg3PDjaR+Mqf9w6bZNm6Sr2LCJrxTLr71PcpZC0nD0menvUkAzwe4
+BzVmciSZWtyQB0fhlr6cBGb0WpqgYlXOV0TecMtAZGKrzsT48fspeBGPPobW9t6Y
+snFgQQB1V3ON4VxHxDeD3OV9Aq91zLl1cgBmp/z65APzzqHXthX/meBCzKLO06w8
+2Np/gIeksFA05HbbykZElslbB2eFz8W3tV4WLWcKucDPl+Pmzlbt8XprWE7Pyn6m
+Fp8beZQWT0VOcSTH/UOfEImplxFLRDTLk0wjMye/i06XlPu/1nrditHwmlVjFbdc
+7NSiO8rXdUgTuOEwdZMyIhCB9SWNxZa+7F3kVKdXTBytVaYSfD3qoDBP8bhaeDF8
+K6054uo5pmBXD2f8WGqbuikNh64i1oncmj475uxRKkzByrkY9XN9qRKjWav6/Zem
+xMRgGmV+HHef8lhyLthDvucIEHELuRK+xWmcD4fn5Mhk4DN4LLezwokCVAQTAQgA
+PgIbIwUJCWYBgAULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLYAZGC2CoDnggYk
+SedH3x+VdaOqBQJZyM/oAAoJEOdH3x+VdaOqTDEP/imxxu+aGjfXXkHX/UELGHIy
+vOGEOEQ83yplaHwot9x1xMUmo35ifJUqkNg5bLy05RKM8T4h57OqN2XCCGgPmCpF
++tbWDyYHuF63awMiXY+tjrtF/6tijDAouM1jeAY1+lUWurhBsiYXkXX7jgV8Ts+O
+wdJkY7r8TnhltpJnfqDTmS40BAY6aja15cyCTvdqpAUnYl3Vc2rlxeSv4tiUhcK7
+JaK8uLE0cAJkE+KlHwraxtKVLrmza56KRRShxVR97zBbykO9oC4qLHCAg3JpkjPV
+0We7M3cnrBrM3n/O7uDy5QVFlVXEbT7YLHkPTom3gcPr+Yk95jyyDud5DUlwx7/l
+cgV5yc07Mu696oBClv5FAZXRpDCHNppmA+i5Yc09EKBeEKtLcIA6cVdx6DZP/VNS
+qeWuHDSt+59v9XzdS/5EVSpD1EyAWcg4DGWBWVa0FN+FBU8WQApmIBENKuhHpXHa
+5zUDTvWk7QUWypu8BLtr6CT71xG8enX78v1bVkiNgmDetPfEs9eNgREZHO9U3TwQ
+jCPMxFmpBCxTnTKsuYTnXpooiCcScecVE1XdfmwgnlVldVMEb5Lx3NTiFRat3vUE
+4FbfaYfv/4QQEBbDmOKwk/6sb6yW1+HSDCTejISmxEF/cYo+7RsbKpC/+fsNeoL+
+6bFc59IK9uH3abykibs+iQJXBBMBCABBAhsjBQkJZgGABQsJCAcCBhUICQoLAgQW
+AgMBAh4BAheAFiEEtgBkYLYKgOeCBiRJ50ffH5V1o6oFAlnD+lACGQEACgkQ50ff
+H5V1o6qgug//dEKAwRn5Z89OL1vUtulvQVizD2PpX3QleqK78Q58o5NBx/HWwXis
+0Q/2mjokOGxTtnjzZI1suzrhbXXqY4kEmbqlNopLP/lQIwEFs3DMQKAWpvag5E4J
+uMyPFoX/GzkeBAd5Q65zw1f21C+shQvORHqXhlaOk0t6syKTtDXxv06MDa3OtB9r
+Gqmr2LEEwdaRxcir8qX5VggHkZ/2Kr18P3CrV0P2QCNwbmbHpYRHk2DGHjlrXbtQ
+9calHHaLZzkABNilm7qr++hUF3TtP9/UtGJFW+dtACWimKq3SLE/LL1LyOSzaKJd
+DKSqfe8zlVlvIhmctsYpTaslueJT/BRQaQQlyUmWu9cAzWlado83e2Xh2fbvM0C+
+jJDnHrt0gIRiN+SPa0n3DtWeBOT8i47cKoicfxQ9k6++CYG4CmPJO2BBujWq+OCx
+g7B/RWmyD5AKP24NZgjEdW0f4yqSZBO50YU5QeaMcE/2JlOBXQENzTd4TEft4hSP
+OoQe1cqb/3QzV3m3Jtopdl05ly0OpIiYe1R/4GqfEfEMavYgVzR2cAsQm8qTiZGA
+o8CvEE3+FHofE8jtm57Ie9N6+8r6NQwKD0qWxnO11h279pqlvZVjGoFzowRfWplj
+Sc4W78sVY5bS5obfvTRBXg2N0iJfIEfv159j42qOcOpHb4PsPQFzu+yJAlQEEwEI
+ACcFAlgDknYCGyMFCQlmAYAFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AAIQkQ50ff
+H5V1o6oWIQS2AGRgtgqA54IGJEnnR98flXWjqiyQEACJhRx41mwz3E4J7Lm65nDr
+CQ/Y8jCUHi6LzthSR2Icyr+DSiXSb5ngsTZFj/qSGR1GdpQQ5MbOKpiKw04ziDRB
+21tWw+0svijdCd/ZyLIUF4EF0Z4q4JCD65E/bmjzd80+fmLVOnns2eBdAdzw+RxZ
+BP4qojsHsMLTDg6YrPgpwz9leQkJVJKI/OJGymWzSohi+I7a2wjra55+8+2Hy14A
+C7y/LjTSeqSlgyPhkE3/WVr4ou6V4lhR9SRPqIrFhL+s6p/XgiCQ+W5oo+VRGrTF
+NIk+UcbdEPBRzS4+eowGyVT44R5dTTOFUeeAZYG1Cx6f0B3dxDhmwqFIEA8k6mIB
+UsWolv4fT/c+K5uh1OYNXvsw+iKYj/v7oraaY4RpWPbOGMRMgXcC8BQUChbBX/Ik
+0iXTz5o6ViIhsHbU9OT+eOREjtyFEGldVBwrPrModEMy0r6y/HlMMVLugLAUxjuH
+QKiK/GQCk80CzSWczmSuGDDHP2dlYN1KYMhLAwfxe0scES1sXL7MlGD6KtTiM2A5
+bKYENmvIzZ6ae8AY4yM9zpdRgWJnr7dtLDuiIdLbIiLPVhfTlPBzBjmeERZ3ry/f
+J6R1mkG1l3Jfx8Yl7tTFqTZ2UAwNR4GF57pDtNiJO5x6+Oq1QvMoKPm0pJj8wED5
+mNSpt8ngq3DoWy3Na22ASbkCDQRYA5J2ARAAyHww3huLEtsdyqgjiGMhtEKOLmp7
+yFl450HY9oPcHS02U5BC1370ssNShrdOCi2ACDbe41Zxx85WcuaO1OVqung2umX0
+47mj2xQsiTAFRDLZsQu8cQFoEy/DBL2bk7ThfK1Lh+NyZAs0UaPpDkGodS0De9os
+A+4T6Nf4POYaeavbYVFSdDKS4lUboBqApKnD/TzKFxFcpuFx6FN92lteTbOojGMi
+LoZvELY86Kn9KuFZ8FM2ZSNHx1Z75KouufGrdkeCoZYVYiuzT+fnt2it4dIpIlnF
++yxMt5LB/MSrmECB5CAFJtxzuMccm6yDUZQSWWi9vUgxIJwvt5w0CIBT353DGeP4
+WnH0r5YoBKoRbh7i4fT0lWvMXTG/V2lqyzBdClMebyHffMgba26Kj6oeDygDfC5a
+GsVaqw1Ue/qQ5QRqTJcJV7xVLTtS1EamVqkfKwPS0zTfnrF1jQtnO/P4qkfgBRRG
+9BXGGrykHpXOyqmX6Z0wbV2P4j+p02oSecDl5yVXplJfsXfbS/xXnaSkaN/7mCU2
+9ul26cAVNxDkDPunztSFi9K9LM2T/XWYJQGXM71OpmONQJGF24lx7Wp/kobnHtbj
+GDzjDPC4eSL7MA56qtrWaLM+4ePKANct2q0q6c0uSLs0Q2zochS64Mcg0YzL1sin
+WPN1rXLDk3lwpIsAEQEAAYkCWgQYAQgADwUCYWawGAIbDAUJFNHfoQA/CRDnR98f
+lXWjqgkQ50ffH5V1o6oJEOdH3x+VdaOqCRDnR98flXWjqhYhBLYAZGC2CoDnggYk
+SedH3x+VdaOqtBgQALeChWgHEGe/8nwKWrC8CMQyKyiJRlSfvERi5M40PYxwKC7I
+Ho+ekMdlLc8kVkv5WSq3zQcNoRMjHutZlCVpGJat5PwiCine6I3Z5JfKsU7fJE4K
+XPD0jr9kCy/IHlCWsKLZTHH0LOk98tqZcFQwFM5nQEzu8Us86BxlZs5IaN0jILD2
+qUC+EYeg0hd0kya+16UNao9NxCeUEoSwxhtB4IIMCJTUPx0pWErCHxeCUjvxPd2h
+5A106DT/3fa8uhPHr9goHtfreV0soGuxu1rqGrMn7XcorsGdt9XX4tyPHcXDXCZM
+LedjYTu7OcYQ63hi+ZWUXJPvcIrB4StCumaFECwLodpdDWB7n6OF8r+X//ePbjS8
+qt++A20LGD4l7BrWxDeymrnqLmWN8RBf2xV4ytVfcRx1ercFMGiNRUvGSh+kgP6J
+7D4yT4xGTq+fCtS6BHyWMB0loXcWv0Im/6znpIpUcvXyL68s4Jiqukt94rKlwb/I
+B6MSWannxk8UzWfGgFXeGUrR8YWLuQSo26AceNOyu2gMW+k9kYi0sqkBzMPk1zWZ
+N/gSNpDyu7AWGpv4RtOKhJhaux8zwtwqozqSx+plpnYz5ifwCNMuznM6T0j4msKQ
+iK1N7lIwKBGaT2P8SJ/ecaJyudIz4ds2+GUl6fxqSR27Egs+EUY0BmS7wESr
+=4FTb
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0


